dspace-server-webapp/src/main/java/org/dspace/app/rest/BitstreamRestController.java

```diff
@@ -182,4 +202,47 @@ private boolean isNotAnErrorResponse(HttpServletResponse response) {
         return responseCode.equals(Response.Status.Family.SUCCESSFUL)
             || responseCode.equals(Response.Status.Family.REDIRECTION);
     }
+
+    /**
+     * This method will update the bitstream format of the bitstream that corresponds to the provided bitstream uuid.
+     *
+     * @param uuid The UUID of the bitstream for which to update the bitstream format
+     * @param request  The request object
+     * @return The wrapped resource containing the new bitstream format or null when the bitstream format wasn't changed
+     * @throws SQLException       If something goes wrong in the database
+     */
+    @RequestMapping(method = PUT, consumes = {"text/uri-list"}, value = "format")
+    @PreAuthorize("hasPermission(#uuid, 'BITSTREAM','WRITE')")
+    @PostAuthorize("returnObject != null")
+    public BitstreamResource updateBitstreamFormat(@PathVariable UUID uuid,
+                                                   HttpServletRequest request) throws SQLException {
+
+        Context context = obtainContext(request);
+
+        List<BitstreamFormat> bitstreamFormats = utils.constructBitstreamFormatList(request, context);
+
+        if (bitstreamFormats.size() > 1) {
+            throw new DSpaceBadRequestException("Only one bitstream format is allowed");
+        }
+
+        BitstreamFormat bitstreamFormat = bitstreamFormats.stream().findFirst()
+                .orElseThrow(() -> new DSpaceBadRequestException("The bitstream format doesn't exist " +
```

Comment:
the message doesn't seem to match the condition,,, should this mean that no bitstream format has been specified?
the next lines seems to deal with the bitstream format not found scenario